test_models.py

```diff
@@ -0,0 +1,58 @@
+from datetime import datetime
+
+import pytest
+from sqlmodel import Session, create_engine
+
+from models import Category, SQLModel, Transaction
+
+engine = create_engine("sqlite:///:memory:", echo=True)
+
+
+@pytest.fixture(scope="function", autouse=True)
```

Comment:
I think creation of db can be done once? = session scope

Reply:
Yep good point, because as soon as you start to add rows, you either need to roll back which is extra code, then it's easier to just have a clean db for each test, besides in-mem is really fast.